install/kubernetes/helm/istio/charts/ingress/templates/service.yaml

```diff
@@ -1,28 +1,43 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ .Release.Name }}-ingress
+  name: istio-ingress
   labels:
-    app: {{ template "istio.name" . }}
     chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
     istio: ingress
 spec:
-{{- if .Values.service.nodePort.enabled }}
```

Comment:
the nodePort.enabled feature is to permit people to eval if they lack a load balancer.  I think the current istio generated manifest has this nodeport feature commented out.  I think the conditional is warranted.

Reply:
Super useful for minikube - but instead of conditional I added 2 services, one with node port 
and one load balancer. This way we can have both.

Reply:
wfm.